build/geany-plugins.nsi

```diff
@@ -401,6 +398,12 @@ Function .onInit
 		StrCpy $INSTDIR "$GEANY_INSTDIR"
 	${EndIf}
 
+	; if $INSTDIR has not been set yet above, set it to the profile directory for non-admin users
+	${If} $INSTDIR == ""
```

Comment:
The diff is a bit cluttered because of the variable rename (I don't know what was wrong with me naming it "Answer" :( ), the relevant part is here: we set `$INSTDIR` only to the user's profile directory if we didn't find a Geany installation before. Before the Geany installation was ignored if the installer was executed without administrative privileges.